[Bug] "unexpected end of file - ; expected" when trying to iterate over loadedMods

#### Issue Description:
Trying to get a list of all mods doesn't work.


#### What happens:
`[INITIALIZATION][CLIENT][ERROR] :1 > unexpected end of file - ; expected`


#### What you expected to happen:
List all mods in the log.


#### Script used:
[script file](https://pastebin.com/FVhfAdqT)

```
import crafttweaker.mods.ILoadedMods;
 
print("listing mods...");
 
for mod in loadedMods {
    print( mod.id ~ " + " ~ mod.name );
}
 
print("done!");
```


#### crafttweaker.log file:
[crafttweaker.log](https://pastebin.com/S69JK2NG)

____
#### Affected Versions:

- Minecraft: `1.12.2`
- Forge: `14.23.5.2776`
- Crafttweaker: `4.1.15`
- Using a server: no

--- a/src/main/java/stanhebben/zenscript/type/ZenType.java
+++ b/src/main/java/stanhebben/zenscript/type/ZenType.java
@@ -104,9 +104,9 @@ public abstract class ZenType implements IDumpConvertable {
     
     public static ZenType parse(String type, IEnvironmentGlobal environment) {
         try {
-            final String fileName = "";
+            final String fileName = "Internal_ZenTypeParse";
             ZenTokener parser = new ZenTokener(type, environment.getEnvironment(), fileName, false);
-            parser.setFile(new ZenParsedFile(fileName, fileName, parser, environment));
+            parser.setFile(new ZenParsedFile(fileName, fileName, new ZenTokener("", environment.getEnvironment(), fileName, false), environment));
             return read(parser, environment);
         } catch(IOException ex) {
             return null;

